src/regulations/NationalBankRegulations.java

```diff
@@ -0,0 +1,49 @@
+package regulations;
+
+import java.util.*;
+
+public final class NationalBankRegulations {
+    private final static Map<String, String> allowedCurrencies;
+    private final static String[] networkProcessors;
+    private static final double maxThresholdDebtRatioIndividual;
+    private static final double maxThresholdDebtRatioCompany;
+
+    static {
+        allowedCurrencies = new HashMap<String, String>();
```

Comment:
no need to write String, String, use the <> diamond operator new HashMap<>() - it infers the types from the left side on line 6

Reply:
I know that new HashMap<>() also work, but I do not like the compiler to decide for me :D. 
For etapa2, whenever I will use again data structures, I will use the diamond operator as you suggested